Core/Rewriters/RewriterBase.cs

```diff
@@ -20,8 +20,19 @@ namespace RhinoMocksToMoqRewriter.Core.Rewriters
 {
   public class RewriterBase : CSharpSyntaxRewriter
   {
-    public SemanticModel? Model { get; set; }
+    private SemanticModel? _model;
+    private SyntaxGenerator? _generator;
 
-    public SyntaxGenerator? Generator { get; set; }
+    public SemanticModel Model
```

Comment:
The semantic model change should be done before this PR with all usages adapted.